mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -457,8 +456,10 @@ public void setMobile(String mobile) {
 
     @Override
     public void setImage(String fullName) {
-        TextDrawable drawable = TextDrawable.builder().buildRound(fullName.substring(0, 1),
-                R.color.colorPrimary);
+        TextDrawable drawable = TextDrawable.builder().beginConfig()
```

Comment:
Btw. I had to change the way how TextDrawable is building, because of blank space error while was trying to set a TextDrawable as an ImageDrawable to the CircleImageView. Adding height and width fix the problem.